cmd/install/install.go

```diff
@@ -480,6 +488,43 @@ func hyperShiftOperatorManifests(opts Options) ([]crclient.Object, error) {
 		objects = append(objects, userCABundleCM)
 	}
 
+	if mgmtClusterCaps.Has(capabilities.CapabilityImage) && mgmtClusterCaps.Has(capabilities.CapabilityProxy) {
+		imageRegistryCABundle, err := getImageRegistryCABundle()
```

Comment:
This returns a string but you then assign a ConfigMap pointer on line 501. I would suggest declaring a var for your ConfigMap here and create a new variable to capture the string returned from `getImageRegistryCABundle`.

Reply:
Thanks for catching this! Modified it so that they're under different variable names.